### ISSUE TITLE
VS Code capturing capslock key

### ISSUE THREAD

**Issue by chrisnorman7** [microsoft/vscode#178620]
Type: <b>Bug</b>

Hi,
I am using VS Code with a screenreader (NVDA to be precise), and I cannot use the capslock key with NVDA when inside VS Code.

This issue doesn't seem to affect the insiders build, so maybe this has been fixed already.

Thanks in advance.

VS Code version: Code 1.76.2 (ee2b180d582a7f601fa6ecfdad8d9fd269ab1884, 2023-03-14T17:55:54.936Z)
OS version: Windows_NT x64 10.0.22621
Modes:
Sandboxed: Yes

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 9 5950X 16-Core Processor             (32 x 3400)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|31.91GB (22.55GB free)|
|Process Argv|--crash-reporter-id 8e8a2fa2-86e5-4e51-a956-4e42704243aa|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (75)</summary>

Extension|Author (truncated)|Version
---|---|---
odin|aes|1.2.2
Bookmarks|ale|13.3.1
flutter-snippets|ale|3.0.0
tsl-problem-matcher|amo|0.6.2
code-gnu-global|aus|0.2.2
better-toml|bun|0.3.2
npm-intellisense|chr|1.4.4
ols|Dan|0.1.13
dart-code|Dar|3.61.20230324
flutter|Dar|3.60.0
vscode-markdownlint|Dav|0.49.0
jshint|dba|0.11.0
vscode-eslint|dba|2.4.0
xml|Dot|2.5.1
gitlens|eam|13.4.0
vscode-npm-script|eg2|0.3.29
vscode-pull-request-github|Git|0.60.0
go|gol|0.38.0
rails-snippets|Hri|1.0.8
better-cpp-syntax|jef|1.17.2
cmake-language-support-vscode|jos|0.0.9
indentation-level-movement|kai|1.2.1
restructuredtext|lex|189.3.0
coffeelinter|lky|1.4.0
flutter-intl|loc|1.19.1
build-runner|Luk|0.8.1
bash-ide-vscode|mad|1.36.0
vscode-prototool|mat|0.2.0
vscode-docker|ms-|1.24.0
csharp|ms-|1.25.4
vscode-dotnet-runtime|ms-|1.6.0
isort|ms-|2023.9.10861010
python|ms-|2023.5.10791008
vscode-pylance|ms-|2023.3.31
jupyter|ms-|2023.2.1200692131
jupyter-keymap|ms-|1.1.0
jupyter-renderers|ms-|1.0.15
vscode-jupyter-cell-tags|ms-|0.1.8
vscode-jupyter-slideshow|ms-|0.1.5
remote-containers|ms-|0.282.0
remote-ssh|ms-|0.98.0
remote-ssh-edit|ms-|0.84.0
remote-wsl|ms-|0.76.1
azure-account|ms-|0.11.3
cmake-tools|ms-|1.13.45
cpptools|ms-|1.15.1
cpptools-extension-pack|ms-|1.3.0
remote-explorer|ms-|0.2.0
vscode-serial-monitor|ms-|0.9.0
wordcount|ms-|0.1.0
vsliveshare|ms-|1.0.5834
vsliveshare-audio|ms-|0.1.93
vsliveshare-pack|ms-|0.4.0
gradle-language|nac|0.2.3
ruby|reb|0.28.1
vscode-commons|red|0.0.6
vscode-xml|red|0.24.0
vscode-yaml|red|1.12.2
flutter-riverpod-snippets|rob|1.2.2
rust-analyzer|rus|0.3.1451
jumptocolumn|sal|1.1.0
rewrap|stk|1.16.3
code-spell-checker|str|2.20.3
odin|the|0.0.2
indent-nav|Ton|1.0.3
simple-rst|tro|1.5.2
cmake|twx|0.0.17
intellicode-api-usage-examples|Vis|0.2.7
vscodeintellicode|Vis|1.2.30
vscode-arduino|vsc|0.6.0
vscode-wakatime|Wak|24.0.7
vscode-ruby|win|0.28.0
unsaved-files-vscode|wra|2.1.3
markdown-all-in-one|yzh|3.5.1
vscode-proto3|zxh|0.5.5

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30445986
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vscaac:30438847
vsclangdf:30486550
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30695312
azure-dev_surveyone:30548225
nodejswelcome1cf:30587006
2e4cg342:30602488
pyind779:30671433
f6dab269:30613381
pythonsymbol12:30671437
vsctsb:30677850
azdwalk:30687957
pythonms35:30686772

```

</details>
